nri-config-generator/tests/generator_test.go

```diff
@@ -36,6 +37,10 @@ const configPDNSTemplate = `
         "config": {
           "standalone": false,
          {{ or .pomiVerbose "" }}
+		  "integration_metadata":{
+			"name": "powerdns",
+			"version": "test-tag"
+		  },
```

Comment:
could your reformat to spaces? indentation looks off

Reply:
true, my Ide was showing tabs in a different way for some reason 😖